autogen/agentchat/contrib/multimodal_web_surfer/multimodal_web_surfer.py

```diff
@@ -411,9 +417,17 @@ def generate_surfer_reply(
                 percent_visible=percent_visible,
                 percent_scrolled=percent_scrolled,
             )
+
+        if IS_OCR_CAPABLE:
+            ocr_text = self._get_ocr_text(new_screenshot)
+        text_prompt = f"""Here is a screenshot of [{self._page.title()}]({self._page.url}).
+            The viewport shows {percent_visible}% of the webpage, and is positioned {position_text}."""
+        if ocr_text is not None:
+            text_prompt += ocr_text
```

Comment:
We might consider adding some preamble like:

"The text content of the viewport is:\n"+ ocr_text